src/Host.cpp

```diff
@@ -395,6 +394,7 @@ Host::Host(int port, const QString& hostname, const QString& login, const QStrin
 , mPlayerRoomInnerDiameterPercentage(70)
 , mDebugShowAllProblemCodepoints(false)
 , mCompactInputLine(false)
+, mMapInfoContributors(QSet<QString>{"Short"})
```

Comment:
I think we should change the default to be simpler; user feedback has shown this is a welcome change

Reply:
Well, my previous save - like other users - had the info display ON - so it was showing the "Full" display. For the display to switch to something else without warning - it turned out to be the "full" and the "brief" one and I had to intervene to get what I wanted back. During the evaluation of this PR this removes the old setting and puts something that the current *release* version is not going to understand in instead. So it will - I think remove my old-style "info on" setting from the save file and thus it will reload in the release version with the info display on (because that is the default in the current code). Now if I were someone who normally played with the info display off then that would not be preserved if I tried this in, say a PTB.

For compatibility during the change over. If this new version reads the old info setting (`"mShowInfo"`) in the save file (and does not get anything for the new setting) and it is `"no"` then this option should be cleared - set to the value that corresponds to no display ("none" isn't it?). When saving, if the option is "none" then this new version should also write the old setting with the "no" setting - but if it is anything else it should not write anything for the old setting. This will ensure that during the change over the save file is still as much compatible with older Mudlet versions as it can be.

Reply:
> I think we should change the default to be simpler; user feedback has shown this is a welcome change

How about putting the default in new profiles but keep existing ones unchanged?

Reply:
@Kebap Yeah, I think this is good idea.

Reply:
I still think this is a change most users would benefit from, and those few who'd like to stick with Full - they can. Do we have anyone else who's mentioned they like the verbose display besides @SlySven?

This is about the current mass of users getting an improvement right away vs a few users having to change the option back (which is **trivial**) and I suspect it's more beneficial for the majority to benefit.

Reply:
Maybe some developer poll? Discord users poll? :) Not sure what to do here :)

Reply:
Yeah we can do a poll among the Patreon supporters, one of the perks they get to do!

Reply:
The point of https://github.com/Mudlet/Mudlet/pull/4683 is to apply it for everyone, and we also have GUI makers on Discord wanting this:

![image](https://user-images.githubusercontent.com/110988/106604830-763e6b00-6560-11eb-9e85-3413682fd1b8.png)

Reply:
I am concerned about a smooth switch over - a current user who plays with the info display off and will have that setting saved in their current profiles will, if they try a PTB with this code in, suddenly start getting the brief display in their mapper without being asked. If they then save their profile and go back to the current release version the old setting is not saved so when they reload in that they will get the old full info (because the default in the absence of the *old* setting being saved in the file is to show the full info)...

So as to **ensure a smooth transition whilst this is evolving** this code should continue to save the "mShowInfo" with the "no" value if the user leaves the profile with no infos being shown - otherwise the old setting ***can be left out***. This gives the most consistent behaviour yet respecting the end-user's choice; this is not too much to ask, surely?

Reply:
Hmm that's different subject than what we're talking about.

>  this code should continue to save the "mShowInfo" with the "no" value if the user leaves the profile with no infos being shown 

Agree

Reply:
Actually saving old map info seems pretty trivial, so there it is. I've added updating its state.

And regarding the main point of discussion... so maybe we can try out this pole, and hear what people think :)

Reply:
I think at the end of the day I got @SlySven convinced that everyone thinks it's a good idea. I understand the importance of consistency but in this instance the inconsistency incurred for an improvement people will appreciate is a win